tests/common/utilities.py

```diff
@@ -486,7 +486,10 @@ def compare_crm_facts(left, right):
     for k, v in list(left['resources'].items()):
         lv = v
         rv = right['resources'][k]
-        if lv['available'] != rv['available'] or lv['used'] != rv['used']:
+
+        # For Cisco-8000 devices, hardware counters are statistical-based with +/- 1 entry tolerance.
+        # Hence, the available counter may not increase as per initial value.
```

Comment:
not quite understand, could you explain more?

Reply:
* Available entries does not ALWAYS decrease/increase on adding/deleting a route. Case to case where the entry added to an existing bucket will not effect the numbers.
* For an example if there are 100 available routes, it is not guaranteed to add 100 prefixes. When each prefix lands in different set of buckets.

So, we have taken +/- 1 entry tolerance.